src/pages/index.js

```diff
@@ -95,14 +171,66 @@ const editProfileModal = new PopupWithForm(
 // Set up event listeners
 addCardButton.addEventListener("click", () => addCardModal.open());
 editProfileButton.addEventListener("click", () => {
-  const { job, name } = userInfo.getUserInfo(); // get the data
-  // here you'll insert the job, name into the inputs value
+  const { job, name } = userInfo.getUserInfo();
+  profileNameInput.value = name;
+  profileJobInput.value = job;
   editProfileModal.open();
 });
 
 editProfileModal.setEventListeners();
 addCardModal.setEventListeners();
+confirmPopup.setEventListeners();
+profileImageModal.setEventListeners();
+
+//The Accursed API
+const api = new API({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "c371b666-258b-4f19-aeb9-028c93427d7f",
+    "Content-Type": "application/json",
+  },
+});
 
-//dev note for reviewer below
+//The Foul Spawn of the API Demon
+function handleError(err) {
+  console.error("Request failed:", err);
+}
 
-// sorry about all of the resubmitions! I have been busy with college as this week was finals week. I thought maybe id need to send you the node_modules folder too to make my code work because I was not getting the error you were and all my modals worked.
+api
+  .getInitialCards()
+  .then((cards) => {
+    cardSection.renderItems(cards);
+  })
+  .catch(handleError);
+
+api
+  .getUserInfo()
+  .then((data) => {
+    userInfo.setUserInfo({
+      name: data.name,
+      job: data.about,
+      avatar: data.avatar,
+    });
+    userInfo.setUserInfo(data.avatar);
+  })
+  .catch(handleError);
+
+function handleLikeClick(card) {
+  if (card._isLiked) {

```

Comment:
`_isLiked`

Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.